frappe/print/page/print_format_builder/print_format_builder.js

```diff
@@ -417,6 +418,40 @@ frappe.PrintFormatBuilder = Class.extend({
 			return false;
 		});
 	},
+	setup_field_settings: function() {
+		var me = this;
+		this.page.main.on("click", ".field-settings", function() {
+			var field = $(this).parent();
+// 			var no_of_columns = section.find(".section-column").length;
+
+			// new dialog
+			var d = new frappe.ui.Dialog({
+				title: "Set Alignment",
+				fields: [
+					{
+						label: __("Align"),
+						fieldname: "align",
+						fieldtype: "Select",
+						options: [{'label': __('Left'), 'value': 'left'}, {'label': __('Right'), 'value': 'right'}]
+					}
+				],
```

Comment:
@KanchanChauhan let's also allow the user to change the Label - its a common use-case, its the same workflow as align (will add it to the format json)